types/wordpress__components/flex/index.d.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentType, HTMLProps, CSSProperties } from 'react';
+
+declare namespace Flex {
+
+    interface FlexProps {
+        /**
+         * Sets align-items. Top and bottom are shorthand for flex-start and flex-end respectively.
+         */
+        align?: CSSProperties['alignItems'];
```

Comment:
Thanks for the link (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47381#discussion_r486513348). I think it was a good idea to include them in the type.

```suggestion
        align?: CSSProperties['alignItems'] | 'top' | 'bottom';
```